content/_code-samples/delete-account/py/delete-account.py

```diff
@@ -0,0 +1,62 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models.transactions import AccountSet, SetRegularKey, AccountSetFlag, Payment
+from xrpl.transaction import safe_sign_and_autofill_transaction, send_reliable_submission
+from xrpl.wallet import generate_faucet_wallet
+from xrpl.models.requests import AccountInfo
+
+# This code sample will blackhole an account
+# A funded account on the testnet is provided for testing purposes
+# https://xrpl.org/accounts.html#special-addresses
+
+asfDisableMaster = 1114112
+
+# Connect to a testnet node
+JSON_RPC_URL = "https://s.altnet.rippletest.net:51234/"
+client = JsonRpcClient(JSON_RPC_URL)
+
+# Get credentials from the Testnet Faucet
+print("Requesting account from the Testnet faucet...")
+test_wallet = generate_faucet_wallet(client=client)
+myAddr = test_wallet.classic_address
+
+print(f"\n Account: {test_wallet.classic_address}")
+print(f"    Seed: {test_wallet.seed}")
+
+# Construct SetRegularKey transaction
+tx_regulary_key = SetRegularKey(
+    account=myAddr,
+    regular_key="rrrrrrrrrrrrrrrrrrrrBZbvji"
+)
+
+# Sign the transaction
+tx_regulary_key_signed = safe_sign_and_autofill_transaction(tx_regulary_key, wallet=test_wallet, client=client)
+submit_tx_regular = send_reliable_submission(client=client, transaction=tx_regulary_key_signed)
+submit_tx_regular = submit_tx_regular.result
+print(f"\n SetRegularKey tx submit result: {submit_tx_regular['meta']['TransactionResult']}")
```

Comment:
(This change will change how many spaces the next line should have to be lined up)
```suggestion
print(f"\n Submitted a SetRegularKey tx.  Result: {submit_tx_regular['meta']['TransactionResult']}")
```